addons/account/__init__.py

```diff
@@ -14,10 +15,87 @@ def _account_post_init(env):
     _set_fiscal_country(env)
 
 # imported here to avoid dependency cycle issues
-# pylint: disable=wrong-import-position
 from . import controllers
 from . import models
 from . import demo
 from . import wizard
 from . import report
 from . import tools
+from .demo.account_demo import AccountChartTemplate
+from .models.account_account import AccountAccount, AccountGroup
+from .models.account_account_tag import AccountAccountTag
+from .models.account_analytic_account import AccountAnalyticAccount
+from .models.account_analytic_distribution_model import AccountAnalyticDistributionModel
+from .models.account_analytic_line import AccountAnalyticLine
+from .models.account_analytic_plan import AccountAnalyticApplicability
+from .models.account_bank_statement import AccountBankStatement
+from .models.account_bank_statement_line import AccountBankStatementLine, AccountMove
+from .models.account_cash_rounding import AccountCashRounding
+from .models.account_code_mapping import AccountCodeMapping
+from .models.account_full_reconcile import AccountFullReconcile
+from .models.account_incoterms import AccountIncoterms
+from .models.account_journal import AccountJournal, AccountJournalGroup
+from .models.account_journal_dashboard import AccountJournal
+from .models.account_lock_exception import AccountLock_Exception
+from .models.account_move import AccountMove
+from .models.account_move_line import AccountMoveLine
+from .models.account_move_line_tax_details import AccountMoveLine
```

Comment:
```suggestion
from .models.account_move_line import AccountMoveLine
from .models.account_move_line_tax_details import AccountMoveLine
```
Both files are showing AccountMoveLine. Something is failing here.

Reply:
it's not a bug, the addon is like that. They are an inherent in 2 files